packages/babel-helper-fixtures/src/index.js

```diff
@@ -101,18 +101,31 @@ export default function get(entryLoc): Array<Suite> {
     }
 
     function push(taskName, taskDir) {
-      const actualLoc = findFile(taskDir + "/input");
-      const expectLoc = findFile(taskDir + "/output", true /* allowJSON */);
+      const taskDirStats = fs.statSync(taskDir);
+      let actualLoc = findFile(taskDir + "/input");
       let execLoc = findFile(taskDir + "/exec");
 
+      // If neither input nor exec is present it is not a real testcase
+      if (taskDirStats.isDirectory() && !actualLoc && !execLoc) {
+        return;
```

Comment:
I had this in but then removed it, because having empty directories is not a bad thing. But now that I think of it we should warn if the folder is not empty. I will add that.